### PR TITLE
Add example tests for Breeze2

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Comment by potiuk** [comment]
Hey @Bowrna - as promised (little delayed) - here are example unit tests for Breeze. I am not sure yet how you'd feel about adding some more of thsose, but it woudl be great if in one of the next iterations at least some of the tests will be added so that you learn how to do it. I am happy to discuss it in a bit more detail when you are ready and explain how they work.

File: dev/breeze/src/airflow_breeze/breeze.py [hunk 1]
```diff
@@ -69,12 +69,14 @@ def find_airflow_sources_root():
     os.chdir(__AIRFLOW_SOURCES_ROOT)
 
 
+find_airflow_sources_root()
```
Comment (potiuk) [inline dev/breeze/src/airflow_breeze/breeze.py:72]: BTW. This something we will have to add later - but I will have to think how to do it best. this is just to get the example tests working.

Comment (Bowrna) [reply dev/breeze/src/airflow_breeze/breeze.py:72]: Could you explain more about this part @potiuk ? I couldn't understand it.

**Comment by Bowrna** [comment]
> Hey @Bowrna - as promised (little delayed) - here are example unit tests for Breeze. I am not sure yet how you'd feel about adding some more of thsose, but it woudl be great if in one of the next iterations at least some of the tests will be added so that you learn how to do it. I am happy to discuss it in a bit more detail when you are ready and explain how they work.

I will look into this Draft PR @potiuk .

**Pull request closed without merging**